src/common/helpers/export/export-html.test.js

```diff
@@ -2,6 +2,13 @@ import { generateHtmlBuffer } from './export-html.js'
 
 /** @import {TextAnswer} from '../data-extract/data-extract.js' */
 
+jest.mock('fs', () => ({
+  readFileSync: jest.fn().mockReturnValue('/* Mocked CSS content */')
+}))
```

Comment:
Since this is a static resource on the filepath we're loading, we could choose not to mock these & allow the file to be loaded and included in the snapshot. 

That test would both be simpler and more powerful (e.g. it shows us we genuinely got the path correct & that the css will be loaded)